theme/apache/templates/base.html

```diff
@@ -26,6 +26,19 @@
   <meta name="msapplication-config" content="/favicons/browserconfig.xml">
   <meta name="theme-color" content="#282661">
 
+  <meta property="og:url" content="https://www.apache.org/">
+  <meta property="og:type" content="website">
+  <meta property="og:title" content="{{ page.title }}">
+  <meta property="og:description" content="Home page of The Apache Software Foundation">
```

Comment:
This `base.html` is used as template for multiple pages besides the home page. Maybe this description can be made more generic?

Reply:
I have added the support to give descriptions from individual pages just like title. I have also put a description in siteindex page in-order to test it. @rlenferink if you could update the branch.